src/common/thumbnail-manager.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import {Utils, ThumbnailInfo, MediaType} from '@playkit-js/playkit-js';
+import evaluate from './utils/evaluate';
+
+const DefaultThumbnailConfig: Object = {
+  thumbsWidth: 164,
```

Comment:
what are that numbers? 164,92,100? there are multiple options? I also see height doesn't exist in the previous implementation. can't you get a list of thumbnails?

Reply:
Those are the default values we are using with the thumbnail service (see [here](https://developer.kaltura.com/api-docs/Engage_and_Publish/kaltura-thumbnail-api.html)).
The height (92) is also the default value we are using, but I added also its definition because I need it as part of the calculation when returning `ThumbnailInfo`.

Reply:
I think I'm missing something cause I didn't find those default values, also I see you have an option to request different parameters from the thumbnail so I wonder why it's not open for user selection?

Reply:
You can see the defaults values currently [here](https://github.com/kaltura/kaltura-player-js/blob/master/src/common/utils/thumbs.js#L5) and [here](https://github.com/kaltura/kaltura-player-js/blob/master/src/common/utils/thumbs.js#L6).
But I think you're a bit deviated from the PR subject. This is not a discussion about how the thumbnail service is implemented - this is the way we supported it from player's day one. I've just copied the kaltura thumbnail logic to a new file. If you want to open it to discussion let's setup a team meeting.